# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/BytecodeUtil.cpp
# Commit: d9c1e44eb0ad
# Full Hash: d9c1e44eb0addfc3e44ae299c86381128aa08e7f
# Author: Jan de Mooij <jdemooij@mozilla.com>
# Date: 2019-12-15 21:49:48
# Regressor Bug: 1601599
# File Overlap Count: 1
# Description:
#   Bug 1601599 part 4 - Support Ion OSR at all loops. r=tcampbell
#   
#   Depends on D56700
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D56701
# ==============================================================================

diff -r 93842c3971f8 -r d9c1e44eb0ad js/src/vm/BytecodeUtil.cpp
--- a/js/src/vm/BytecodeUtil.cpp	Sun Dec 15 11:33:36 2019 +0000
+++ b/js/src/vm/BytecodeUtil.cpp	Sun Dec 15 11:34:34 2019 +0000
@@ -1581,8 +1581,8 @@
       break;
 
     case JOF_LOOPHEAD:
-      if (!sp->jsprintf(" (ic: %u, data: %u,%u)", GET_ICINDEX(pc),
-                        LoopHeadCanIonOsr(pc), LoopHeadDepthHint(pc))) {
+      if (!sp->jsprintf(" (ic: %u, depthHint: %u)", GET_ICINDEX(pc),
+                        LoopHeadDepthHint(pc))) {
         return 0;
       }
       break;